src/Classes/ItemsTab.lua

```diff
@@ -2011,6 +2014,22 @@ function ItemsTabClass:EditDisplayItemText(alsoAddItem)
 	main:OpenPopup(500, 500, self.displayItem and "Edit Item Text" or "Create Custom Item from Text", controls, nil, "edit")
 end
 
+-- Opens trade site link for selected item
+function ItemsTabClass:OpenTradeLink()
+	local item = self.displayItem.rawLines
+	local tradeQuery = new("TradeQueryCurItem",item)
```

Comment:
Don't use the raw lines here, as they can be wildly unpredicatable.  `Item` is a proper class in PoB, which probably has all the information you're looking for as properties on the class.

Reply:
done, useing item.explicitModLines now